python/bin/amici_import_petab.py

```diff
@@ -152,44 +158,41 @@ def get_fixed_parameters(condition_file_name, sbml_model,
     return fixed_parameters
 
 
-def constant_species_to_parameters(sbml_model):
-    """Convert constant species in the SBML model to constant parameters.
-
-    This can be used e.g. for setting up models with condition-specific
-    constant species for PEtab, since there it is not possible to specify
-    constant species in the condition table.
+def species_to_parameters(species_ids: list[str],
+                          sbml_model: 'libsbml.Model') -> None:
+    """Turn a SBML species into parameters and replace species references
+    inside the model instance.
 
     Arguments:
-        sbml_model: libsbml model instance
+        species_ids: List of SBML species ID to convert to parameter and ID
```

Comment:
```suggestion
        species_ids: List of SBML species IDs to convert to parameter and ID
```
don't understand this sentence

Reply:
Continued on the next line. A bit unfortunate how it is presented here ;)

Reply:
I got that ;) what is "ID of the resulting parameter"? Means the parameter gets the same ID? (and: I assume you check that such a parameter does not yet exist?)

Reply:
Yes, species is removed and parameter is created with the same ID.
There can't be a species and parameter of the same name, so no need to check.